docs/team/README.md

```diff
@@ -35,6 +35,7 @@ Members include Project Management Committee members and committers. The List in
 ### Committer
 | Name |Apache ID|
 |:---|:--|
+|Zhusheng Xu| aderm |
```

Comment:
This is alpha-beta order, you are not the first one, please keep the right order.

Reply:
ok